4.2 Updates (#140)
OWASP/www-project-web-security-testing-guide: index.md

## Versioned Releases

> Only [v4.2](v42/) is currently available as a web-hosted release. Previous releases are available as PDFs on the **Release Versions** tab.

Author: Drop "only"...

> Only [v4.2](v42/) is currently available as a web-hosted release. Previous releases are available as PDFs on the **Release Versions** tab.

Author: ```suggestion
[v4.2](v42/) is currently available as a web-hosted release. Previous releases are available as PDFs and in some cases web content via the **Release Versions** tab.
```
